### ISSUE TITLE
Changer de propriétaire

### ISSUE THREAD

**Issue by babijo** [YanshuoH/NOMP-Web#196]
Hello à vous deux.
@mrstrlc-yipeng 
@YanshuoH 
Pouvez-vous me mettre en propritaire de ce repository?
Merci

**Comment by YanshuoH** [comment]
Messieurs,

Il faudrait attendre jusqu'à la fin du mois car je suis actuellement en vacances.

Si c'est urgent urgent, je pourrais filer mon mdp à Mister Yipeng

发自我的 iPhone

**Comment by YanshuoH** [comment]
@babijo Je n'arrive pas à te mettre en propriétaire car tu possède déjà un projet "NOMP-Web"

**Comment by babijo** [comment]
hello
comment vas-tu?
Ah bon? je ne sais pas... il faudrait que je regarde. Mais toi de ton côté
est-ce que tu ne peux pas changer le Nom avant de me passer en proprio?
BB

2015-10-28 23:06 GMT+01:00 YANSHUO HUANG notifications@github.com:

> @babijo https://github.com/babijo Je n'arrive pas à te mettre en
> propriétaire car tu possède déjà un projet "NOMP-Web"
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/YanshuoH/NOMP-Web/issues/196#issuecomment-152012362.

## 

## 

BIRREGAH Babiga, Phd
Enseignant-Chercheur/ Assistant professor
ICD UMR-CNRS 6281 (Université de Technologie de Troyes)

Joint Research Unit in Sciences and Technologies for Risk Management
Department of Operational Research, Applied Statistics and Simulation
UTT - ROSAS LM2S H107
12 Rue Marie Curie
CS 42060
10004 TROYES CEDEX
Tél. +33 (0) 325715869
Fax. +33 (0) 325715649
QR: http://goo.gl/Et0A4

**Comment by YanshuoH** [comment]
Oui, je peux très bien changer le nom, mais ce n'est pas très propre non? (Ou bien je suis trop exigent lol).

Idéalement si tu n'as pas besoin le "fork" qui représente une copie du projet d'origine on va dire, tu peux très bien le supprimer. Puis je te transfère le mien.

A+

Yanshuo

发自我的 iPhone
